Lib/typing.py

```diff
@@ -87,6 +87,7 @@
     'Set',
     'FrozenSet',
     'NamedTuple',  # Not really a type.
+    'TypedDict',
```

Comment:
Maybe also add the same comment? You can't use it in `isinstance()` for example.